src/main/java/utils/EditArticleUtils.java

```diff
@@ -0,0 +1,29 @@
+package utils;
+
+import models.Article;
+import service.ArticleService;
+
+import javax.servlet.http.HttpServletRequest;
+
+public class EditArticleUtils {
+
+    public static boolean editArticle(HttpServletRequest request) {
```

Comment:
I propose to convert the signature of this method to `Article parseArticle(HttpServletRequest request)` that returns parsed article and after that ArticleServlet calls `articleService.editArticle(editArticle)`. In this case it will be a real utility method that extracts Article form http request